src/features/components/templates/navigation/career-nav/index.tsx

```diff
@@ -9,13 +9,18 @@ import { Localize } from 'components/localization/localize'
 import LanguageSwitcher from 'features/components/molecules/language-switcher'
 import Link from 'features/components/atoms/link'
 import Flex from 'features/components/atoms/flex-box'
+import { localize } from 'components/localization'
 
 const CareerNav = () => {
     return (
         <NavTemplate
             renderLogo={() => (
-                <Link url={{ type: 'internal', to: '/' }}>
-                    <Image src={CareerNavLogo} className={careers_nav_logo} />
+                <Link url={{ type: 'internal', to: '/' }} aria-label="deriv careers link">
```

Comment:
I'm curious about if the aria-label is not written here will the alt show when there is image loading problem? If yes what is the purpose of aria-label here?

Reply:
aria-label is for screen reader. It will help visually impaired people to navigate website.